CODEOWNERS

```diff
@@ -0,0 +1 @@
+# Submit a helpdesk ticket to add any team other than yours referenced in the CODEOWNERS file -- they must be added to collaborators and teams in the repository settings with maintainer privileges. Remove this file as soon as you have completed this
```

Comment:
Should this contain `@hashicorp/waypoint` like the old file?

Reply:
Yep, I changed that in 4c7e02a